packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -2536,7 +2541,7 @@ export default abstract class SqlIntegration
       columnRef: isFactMetric(metric) ? metric.numerator : null,
     });
     const capCoalesceDenominator = this.capCoalesceValue({
-      valueCol: `m.${alias}_denominator`,
+      valueCol: `${crossFactTableRatio ? "m2" : "m"}.${alias}_denominator`,
```

Comment:
question: What does `m2` or `m` mean here?

Reply:
It corresponds to the alias for the CTE that will need to be used in the final analysis step of the query.